package.json

```diff
@@ -65,6 +68,9 @@
         "default": "./src/style.css"
       }
     },
+    "./style.module": {
+      "sass": "./src/style.module.css"
```

Comment:
Question: when should Sass users import this CSS module?

Reply:
This is actually the export I'm using in my project, and the PR description has some examples.

The strongest example is probably the `@layer` one. 

In my project, I'm using [CSS Cascade Layers](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers) to manage the order of CSS rules written by my team, and dozens of other teams.

Since we use CSS modules and often re-use components, and override styles, we need some way to manage (other than with CSS specificity) the order in which CSS is applied to components. Because we can't predict which order the CSS will load in. 

We've created a dedicated `third-party` Layer which is where we put CSS that comes from third party libraries, which allows us to cleanly re-use third-party CSS, and override it without fighting with its selector's specificity.

I showed the example of how this is being imported in my project


In `MyDatepicker.module.scss`
```scss
@layer third-party {
  @import 'pkg:react-day-picker/styles.module';
}

/* Here we can include any overrides */
.day_button {
  /* Omitted */
}
```

I then import this module in my implementation of the `DayPicker` component.

```tsx
import styles from './MyDatepicker.module.scss';
import { DayPicker } from 'react-day-picker';

const MyDatepicker = () => {
  return <DayPicker {/* omitted */} classNames={styles} />
}
```